package.json

```diff
@@ -7,7 +7,11 @@
     "test:open": "npm run lint && cypress open",
     "test": "cypress run",
     "lint": "eslint ./cypress/e2e/**/*.ts",
-    "lint:fix": "npm run lint -- --fix"
+    "lint:fix": "npm run lint -- --fix",
+    "combine:reports": "mochawesome-merge cypress/reports/mocha/*.json > cypress/reports/report.json",
+    "generate:reports": "marge cypress/reports/report.json -f report -o cypress/reports",
+    "pretest": "rm -r cypress/reports/*",
```

Comment:
pretest should be improved, read more here https://ss64.com/bash/rm.html some flag that might help you if the folder/file does not exist